sci-mathematics/glucas/patches/glucas-2.9.0.patchset

```diff
@@ -0,0 +1,24 @@
+From 2a4d27201ccb87270cd72a0849169d2a22389c0e Mon Sep 17 00:00:00 2001
+From: nishanth1232 <thumma.nishanth79@gmail.com>
+Date: Sat, 17 Nov 2018 18:00:23 +0000
+Subject: gsetup.c for glucas
+
+
+diff --git a/gsetup.c b/gsetup.c
+index 7f32832..794af25 100644
+--- a/gsetup.c
++++ b/gsetup.c
+@@ -39,8 +39,10 @@ void setup (int nonice)
+ #ifdef __sgi__
+     (void)schedctl(NDPRI, 0, NDPLOMIN);
+ #endif
+-#if !defined(macintosh) && !defined(pccompiler)
++#ifndef _HAIKU_
++#if !defined(macintosh) && !defined(pccompiler) && !defined(_HAIKU_)
```

Comment:
It's not `_HAIKU_` but `__HAIKU__`. And you can simplify the bloc a little:
```diff
-#ifndef _HAIKU_
-#if !defined(macintosh) && !defined(pccompiler) && !defined(_HAIKU_)
+#if !defined(macintosh) && !defined(pccompiler) && !defined(__HAIKU__)
     (void)nice(40);
-#endif
 #endif
```

Reply:
For some reason this doesn't do the trick (atleast for me) on x86_64 @fbrosson

Reply:
This is puzzling. (I'll give it a try soon.)